### DISCUSSION TITLE
Missing things that may be actually nice to have

### DISCUSSION THREAD

**Discussion by Paraxdev** [mantinedev/mantine#76] [Feedback]
I Really enjoy the Mantine workflow, but I still have some questions and suggestions:
Missing are:
`Header and
Footer`
Those two are either missing or I can't find nothing in the docs for it.
and my question is: `
  What can be used to perfectly center everything inside a container?`

**Comment by rtivital** [comment]
Hi Dominik, thanks for feedback, Header, Footer and other layout components usually app specific and cannot be generalized to cover all situations, for example, I could export header, footer and navigation just like in Mantine docs, but it would make no sense for most applications. 

> What can be used to perfectly center everything inside a container?

You can use style prop to center things:
```tsx
<div style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>perfectly centered</div>
```

**Reply by Paraxdev** [reply]
yeah header seems useless since drawers are so much better but there is something i don't understand:
![Bug](https://i.imgur.com/Z79sDZ1.gif)

**Reply by Paraxdev** [reply]
the Burger component seems to not set to false? so it somehow reactivates it and i'm clueless.

```javascript
...const [opened, setOpened] = useState(false); 

<div style={{ position: 'relative', zIndex: 1 }}>
                    <ElementsGroup position="left" style={{
                        position:"absolute",
                        top:0,
                        left:0
                    }}>
                        <Burger
                            opened={opened}
                            onClick={() => { setOpened((state: Boolean) => {
                                console.log(`Trying to set ${state} to ${!state}...`);
                                return !state;
                            }); }}
                            title={opened ? "CLOSE NAVIGATION" : "OPEN NAVIGATION"}
                            aria-label={opened ? "CLOSE NAVIGATION" : "OPEN NAVIGATION"}
                        />
                    </ElementsGroup>
                </div>
                ```

**Reply by rtivital** [reply]
It's most likely  drawer registers outside click when burgger is clicked try setting noCloseOnClickOutside  https://mantine.dev/core/drawer/#control-behavior

**Reply by Paraxdev** [reply]
> It's most likely drawer registers outside click when burgger is clicked try setting noCloseOnClickOutside https://mantine.dev/core/drawer/#control-behavior

![Works!](https://i.imgur.com/tX5v3WA.gif)
Yup everything works fine thanks!
